tensorflow/cc/gradients/math_grad_test.cc

```diff
@@ -1091,5 +1091,17 @@ TEST_F(NaryGradTest, Minimum) {
   RunTest(x, x_init_value, y, shape);
 }
 
+TEST_F(NaryGradTest, Lgamma) {
+  TensorShape shape({3, 2});
+  auto x = Placeholder(scope_, DT_FLOAT, Placeholder::Shape(shape));
+  auto y = Lgamma(scope_, x);
+  // Select values to avoid instability when computing finite differences.
+  // Ref: https://en.wikipedia.org/wiki/File:Gamma_plot.svg
+  Tensor x_init_value =
+      test::AsTensor<float>({-3.5f, -2.5f, -1.5f, 1.0f, 2.0f, 3.5f}, {3, 2});
+  RunTest(x, x_init_value, y, shape);
+  // TODO: add test case for complex values
```

Comment:
If you don't plan to add the test case for complex values in this pull request, please assign the todo to me like 'TODO(suharshs): ...' so that someone owns it in the future.

Thanks!

Reply:
modified todo as suggested.